src/pages/project/helpers/projectContributorHelpers.ts

```diff
@@ -15,8 +14,11 @@ export const findProjectManagerIndex = (contributors: ProjectContributor[]) => {
   return contributors.findIndex((contributor) => contributor.roles.some((role) => isProjectManagerRole(role)));
 };
 
-export const getNonProjectManagerContributors = (contributors: ProjectContributor[]) => {
-  return contributors.filter((contributor) => contributor.roles.some((role) => isNonProjectManagerRole(role)));
+export const getContributorsWithRelevantRole = (
+  contributors: ProjectContributor[],
+  roleType: ProjectContributorType
+) => {
+  return contributors.filter((contributor) => contributor.roles.some((role) => role.type === roleType));
```

Comment:
We are now displaying two different sets of users with the same view (LocalManager and ProjectParticipant, ProjectManager has its own view because it has some special cases)